dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/job/TrackerNotificationThread.java

```diff
@@ -66,8 +64,11 @@ public TrackerNotificationThread(
     this.manager = manager;
     this.serviceMapper =
         Map.of(
-            Enrollment.class, programNotificationService::sendEnrollmentNotifications,
-            Event.class, programNotificationService::sendEventCompletionNotifications);
+            SideEffectTrigger.ENROLLMENT, programNotificationService::sendEnrollmentNotifications,
```

Comment:
Do we not react to new/updated events?

Reply:
Only at Enrollment, Enrollment Completion and Event Completion.
Notifications at Event Update can be done using ProgamRule notificaitons.